core/modules/date/views/date_views_argument_handler.inc

```diff
@@ -108,8 +126,10 @@ class date_views_argument_handler extends date_views_argument_handler_simple {
     // There is no easy way to do summary queries on multiple fields, so remove that option.
     unset($defaults['summary']);
 
-    if ($which && !empty($defaults[$which])) {
-      return $defaults[$which];
+    if ($which) {
```

Comment:
Is there a good reason to separate the comparisons here? Not a biggie, just curious. :)